src/iosource/Packet.h

```diff
@@ -7,15 +7,24 @@
 #include <string>
 
 #if defined(__OpenBSD__)
+
 #include <net/bpf.h>
 using pkt_timeval = bpf_timeval;
+#ifndef DLT_RAW
+#define DLT_RAW 14
+#endif
+
 #else
+
 using pkt_timeval = struct timeval;
 #include <sys/socket.h>
 #include <sys/time.h>
+#ifndef DLT_RAW
+// nolintnextline(cppcoreguidelines-macro-usage)
+#define DLT_RAW 12
 #endif
 
-#include <pcap.h> // For DLT_ constants
```

Comment:
I guess the NEWS entry would be good, but also, if someone needs to use `DLT_` constants and didn't have the `<pcap.h>` include, oh well...